mods/d2k/rules/infantry.yaml

```diff
@@ -227,3 +227,16 @@ saboteur:
 	Voiced:
 		VoiceSet: SaboteurVoice
 
+nsfremen:
+	Inherits: fremen
+	Valued:
+		Cost: 0
```

Comment:
This is used by a lot more than just production, so it might be safer to keep this inherited as well?